lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -43,14 +43,26 @@ class Definition
       def initialize(client, response_builder, node_context, index, dispatcher)
         @client = client
         @response_builder = response_builder
+        @node_context = node_context
         @nesting = T.let(node_context.nesting, T::Array[String])
         @index = index
 
-        dispatcher.register(self, :on_call_node_enter)
+        dispatcher.register(self, :on_call_node_enter, :on_symbol_node_enter, :on_string_node_enter)
       end
 
-      sig { params(node: Prism::CallNode).void }
-      def on_call_node_enter(node)
+      sig { params(node: Prism::SymbolNode).void }
+      def on_symbol_node_enter(node)
+        handle_dsl(node) if @node_context.call_node
+      end
+
+      sig { params(node: Prism::StringNode).void }
+      def on_string_node_enter(node)
+        handle_dsl(node) if @node_context.call_node
+      end
```

Comment:
I had that originally actually, but changed it because it felt wrong to call `handle_dsl` before knowing it _was_ a DSL call or not. WDYT about this rename to `handle_possible_dsl`?